### ISSUE TITLE
Test Timeouts with dd4hep > 1.12

### ISSUE THREAD

**Issue by vvolkl** [HEP-FCC/FCCSW#407]
There are some tests that timeout for the "spack stack" test configuration, which uses a newer dd4hep than the lcg configuration.
So far it looks like any job using the forward ecal endcaps gets stuck.

**Comment by gganis** [comment]
This is puzzling ... can tests timing out be run in standalone, so that we can try some profiling?

**Comment by vvolkl** [comment]
Yes, indeed, it is no problem to run it locally. The CI also runs on an lxplus-like centos7 docker image, so it should just be the usual
```
source [init_key4hep.sh|init_lcg.sh]
mkdir build
cd build; make; make test
```

**Comment by vvolkl** [comment]
I did some checks on this and have a bit of a clearer picture. There are actually many issues to fix, so it is good to decouple this a bit. The first thing was to check it with LCG98, which includes a newer dd4hep version. This needs some fixes ( #409  ) but actually still runs the affected tests fine

**Comment by vvolkl** [comment]
I noticed  that the spack stack built geant4 without vecgeom, but I recompiled it with vecgeom and this also doesn't make a difference.

**Comment by vvolkl** [comment]
Fixed by #419
